spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/datastore/DatastoreTransactionManagerAutoConfiguration.java

```diff
@@ -47,11 +49,11 @@
 	@Configuration
 	static class DatastoreTransactionManagerConfiguration {
 
-		private final Datastore datastore;
+		private final Supplier<Datastore> datastore;
```

Comment:
Since we now have `DatastoreProvider`, should we just use it everywhere in place of `Supplier<Datastore>`?

Reply:
Yes good point, fixed.